src/utilcode/debug.cpp

```diff
@@ -349,13 +349,6 @@ bool _DbgBreakCheck(
     STATIC_CONTRACT_FORBID_FAULT;
     STATIC_CONTRACT_DEBUG_ONLY;
 
-    RaiseExceptionOnAssert(rTestAndRaise);
-
-    if (DebugBreakOnAssert())
-    {
-        DebugBreak();
-    }
-
```

Comment:
Ah, I see there is still a check in _DbgBreakCheck() and the other methods funnel to there. My bad, ignore this.